staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -414,6 +414,7 @@ var defaultVersionedKubernetesFeatureGates = map[featuregate.Feature]featuregate
 
 	WatchList: {
 		{Version: version.MustParse("1.27"), Default: false, PreRelease: featuregate.Alpha},
-		{Version: version.MustParse("1.32"), Default: true, PreRelease: featuregate.Beta},
+		// switch this back to false because the json and proto streaming encoders appear to work better.
+		{Version: version.MustParse("1.32"), Default: false, PreRelease: featuregate.Beta},
```

Comment:
same question about whether we'll backport this, and this matching release-1.32 HEAD